hooks/conftest.py

```diff
@@ -11,6 +11,7 @@ def template_files():
         "package-existing-in-manuscript.tex": r"""
     \documentclass{article}
     \usepackage{package-in-manuscript}
+    \newcommand{\command_in_anuscript}
```

Comment:
please see my comment in the load_headers PR.  it is probably better as `\newcommand{\command_in_manuscript}{}`